### ISSUE TITLE
Using CellNet downstream of salmon

### ISSUE THREAD

**Issue by HuynhLab** [pcahan1/CellNet#9]
Thank you for developing CellNet. I am trying to apply CellNet to some of my RNAseq data following your Nature Protocol paper. I have a stable pipeline on my cluster to run salmon, and I was wondering if there's any function within cellnet to take the quant.sf files and go from there. In other words, I was wondering if we can just compute the last part of cn_salmon locally?

**Comment by pcahan1** [comment]
Hi,

This should be possible. You will want to create an object that is similar to what is produced from running cn_salmon. This will entail calling ...

salmon_load_tranEst, which will load the transcript estimates from the quant.sf file.

gene_expr_sum, which will provide gene level extimates

trans_rnaseq, which will normalize by size

Take a look at the cn_salmon function definition so that you make a list with the expected element names

**Comment by rebekabato** [comment]
Hi Patrick! 

I am also using salmon quant.sf files to run CellNet on. I created a big merged file containing all my samples and gene-level estimates (raw read counts). I would like to use trans_rnaseq() to normalize my data, however, I can get around the "total" argument that this function takes. I have read the manual of course, but still don't understand what number I am supposed to use here and how to calculate it. 

When I was googling around at some point found that total=1e5 was applied in this function, but I believe I am supposed to customize this value to my experiment.

**Comment by pcahan1** [comment]
Hi,

If you are using any of our pre-trainined cnProc files, then yes total=1e5 so that your query data is compatible with the training data. If you are generating your own cnProc, then you you should set it to a value that is <= min expected total read count of any of your training or query samples. Thank you for your interest in CellNet. We hope to have an updated and more robust version available soon.

**Comment by rebekabato** [comment]
Thanks for your prompt reply! Would you please elaborate on what kind of normalization happens at this step? I can see that you previously wrote it normalizes by size, but I would love to understand the process a bit more deeply. Or if you could direct me to some online source where I can read about this, that is fine too. Thanks so much!
